src/transforms/uglify.js

```diff
@@ -1,7 +1,5 @@
 const {AST_Node, minify} = require('uglify-js');
 const {toEstree} = require('babel-to-estree');
-const types = require('babel-types');
```

Comment:
Are these `require` statements required for babel? I removed them because the variables are not used.